cluster/credentials.go

```diff
@@ -76,6 +75,11 @@ func createUserCredentials(ctx *Context, tenantShortName string, userdID uuid.UU
 	if err != nil {
 		return err
 	}
+	// create minio postgres configuration for bucket notification
```

Comment:
Shouldn't we create this on a per tenant basis instead?

Reply:
true, done.